floodviz/static/css/mapstyle.css

```diff
@@ -0,0 +1,11 @@
+circle.gage-point{
+    stroke: blue;
+    stroke-width: 2;
+    fill-opacity: 0;
+}
+
+circle.debug-point{
```

Comment:
It is for debug, but it's really nice to be able to see the bounding box on the map, and I'd rather not have to add and remove it all the time. It's disabled by default and doesn't do anything unless you add `debug = True` in `MAP_CONFIG`